core/internal/gethwrappers/generation/compile_contracts.sh

```diff
@@ -2,11 +2,11 @@
 
 echo "compiling contracts"
 
-CDIR=$(dirname "$0")
-COMPILE_COMMAND=$(<"$CDIR/compile_command.txt")
+CDIR="$(dirname "$0")"
+COMPILE_COMMAND="$(<"$CDIR/compile_command.txt")"
```

Comment:
Why add the extra indirection to read from a file?

Reply:
So that it can print the command out if it fails.